tailwind.config.ts

```diff
@@ -8,12 +8,19 @@ const config: Config = {
     ],
     theme: {
         extend: {
+            fontFamily: {
+                body: ['Space Mono', 'monospace'],
+            },
             backgroundImage: {
-                'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
-                'gradient-conic': 'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
+                'blue-gradient':
```

Comment:
- do we need to add this to `taliwind.config` as it will only be used to set the background for the body element?

Reply:
removed add in global CSS file as background